kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -320,6 +365,8 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
           appStateSource,
           appStateContainer)
       }
+
+      runConfigMapDeletion(pod, kubernetesInfo)
```

Comment:
questions:

Does the executors belong to the same app share the configmap?

If that, on executor pod deleted, you will delete the shared the configmap.